basic/notebooks/utils.py

```diff
@@ -0,0 +1,19 @@
+import colorlover as cl
+from IPython.display import HTML
+
+from paths import THEMES_PATH
+import yaml
+import cufflinks as cf
+
+def display_custom_scales():
```

Comment:
I think there is a way to add colorscales directly to plotly library. If you see 'ggplot' cufflinks template, it has a key `'colorscale': 'ggplot'`. So, 'ggplot' is a colorscale defined in plotly and there must be a way to directly add themes to plotly